packages/app/cypress.config.ts

```diff
@@ -52,6 +52,7 @@ export default defineConfig({
       // Delete this as we only want to honor it on parent Cypress when doing E2E Cypress in Cypress testing
       delete process.env.HTTP_PROXY_TARGET_FOR_ORIGIN_REQUESTS
       process.env.CYPRESS_INTERNAL_E2E_TESTING_SELF = 'true'
+      process.env.CYPRESS_INTERNAL_VITE_OPEN_MODE_TESTING = 'true'
```

Comment:
Yes, without this I was getting "cross origin" errors when running some Vite-related system tests. As for your suggestion, I don't see how that would be any different - since we set `CYPRESS_INTERNAL_E2E_TESTING_SELF` to `true` right above it would be functionally equivalent, wouldn't it?

Reply:
Yeah, my suggestions seems weird now I'm re-reading it - line 52 makes this more than clear enough.